zebra/zebra_evpn_mh.c

```diff
@@ -3035,18 +3041,17 @@ static char *zebra_evpn_es_vtep_str(char *vtep_str, struct zebra_evpn_es *es,
 
 	vtep_str[0] = '\0';
 	for (ALL_LIST_ELEMENTS_RO(es->es_vtep_list, node, zvtep)) {
+		char *ip_str = inet_ntop(AF_INET, &zvtep->vtep_ip, ip_buf,
+					 sizeof(ip_buf));
+		if (!ip_str)
+			continue;
+
 		if (first) {
 			first = false;
-			strlcat(vtep_str,
-				inet_ntop(AF_INET, &zvtep->vtep_ip, ip_buf,
-					  sizeof(ip_buf)),
-				vtep_str_size);
+			strlcat(vtep_str, ip_str, vtep_str_size);
```

Comment:
do we actually need two versions of the strlcat?  They look the same to me on either side of the if statement